data/migrations/run_migrations.sh

```diff
@@ -17,7 +17,11 @@ done
 # run updates in parallel. note that we don't bail here, as we want to
 # re-enable the triggers regardless of whether we failed or not.
 for sql in ${migration_dir}/*.sql; do
-    psql -f "$sql" $* &
+    if [[ $sql = *cleanup*.sql ]]; then
```

Comment:
No, because we're using bash's globbing - if we quoted we'd be comparing against a file called `"*cleanup*.sql"`